### PR TITLE
OTC layout 수정

### PR DESCRIPTION
**Pull request by mjkya**

### CODE REVIEW

File: src/components/Target.vue [hunk 1]
```diff
@@ -20,7 +20,7 @@ export default {
     getCurrentTime: (x) => {x.now = Math.floor(new Date().getTime() / 1000)}
   },
 
-  props: ["val", "buslist"],
+  props: ["buslist"],
```
Comment (Taewan-P) [inline src/components/Target.vue:22]: To remove 
image
this into 
image 
this, you should not just delete the properties. You also need to delete the lines that the prop is used.

File: src/components/Target.vue [hunk 2]
```diff
@@ -2,8 +2,8 @@
   <p>
     {{ this.val }}
```
Comment (Taewan-P) [inline src/components/Target.vue]: This is the line that the val is used. If you only delete the properties, it would throw an error over here.

Comment (Taewan-P) [reply src/components/Target.vue]: So if you want the delete the val(bus station name), please delete this line too.

**Review by Taewan-P — CHANGES_REQUESTED** [review]
Except the things that I requested, I think it's good to go.

File: src/components/Target.vue [hunk 3]
```diff
@@ -2,8 +2,8 @@
...
     {{ this.buslist !== "hello" ? hours + "시" : "" }}
```
Comment (CXZ7720) [inline src/components/Target.vue:3]: How about add "v-if" statement to show hours only at midnight?
I think without "hour" is more clear and readable.

Comment (Taewan-P) [reply src/components/Target.vue:3]: I think that is a good idea.

**Review by Taewan-P — APPROVED** [review]
LGTM.

**Pull request merged**